src/core/store/external/state.js

```diff
@@ -3,7 +3,7 @@ const state = {
   darkList: {},
   path: '',
   currencyRate: {},
-  ETHUSDValue: {}
+  coinGeckoTokens: new Map()
```

Comment:
do you mind adding a comment with what the map looks like? it will be easier to use it